lib/screens/repayments/repayments_screen.dart

```diff
@@ -117,6 +134,52 @@ class RepaymentsScreen extends StatelessWidget {
                 log('Repayments analytics');
               },
             ),
+            StoreConnector<AppState, MoreCreditViewModel>(
+              converter: (store) => MoreCreditPresenter.presentMoreCredit(
+                moreCreditState: store.state.moreCreditState,
+                user: user,
+              ),
+              distinct: true,
+              builder: (context, viewModel) {
+                if (viewModel is MoreCreditInitialViewModel) {
+                  StoreProvider.of<AppState>(context).dispatch(
+                    GetMoreCreditCommandAction(
+                      user: user.cognito,
+                    ),
+                  );
+                }
+
+                if (viewModel is MoreCreditLoadingViewModel) {
+                  return const Center(
+                    child: CircularProgressIndicator(),
+                  );
+                }
+
+                if (viewModel is MoreCreditErrorViewModel) {
+                  return const Text('Error loading more credit details');
+                }
+
+                return IvoryListTile(
+                  startIcon: Icons.back_hand_outlined,
+                  title: 'Need more credit?',
+                  subtitle: (viewModel is MoreCreditFetchedViewModel)
+                      ? (viewModel.waitlist == false)
+                          ? 'Sign up for our waitlist'
+                          : 'You\'re on our waitlist'
+                      : 'You\'re on our waitlist',
+                  onTap: () {
+                    (viewModel is MoreCreditFetchedViewModel)
```

Comment:
Cred ca poti sa combini cumva astea doua conditii: (viewModel is MoreCreditFetchedViewModel)+(viewModel.waitlist == false) ca sa fie mai usor de citit codul.